server/controllers/userController.js

```diff
@@ -97,10 +97,41 @@ const register = async (req, res) => {
   }
 };
 
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    // check if user exists
+    const findUser = await findUserByEmail(email);
```

Comment:
Where is this function defined? Also consider an object name as opposed to an action name. When you're using this variable the user will have already been found.